creator/lib/screens/salon_application_screen.dart

```diff
@@ -252,17 +259,25 @@ class _SalonApplicationScreenState extends State<SalonApplicationScreen> {
     );
   }
 
+  sendEmail() {
```

Comment:
Looks like this function is not being used anywhere?

Reply:
I erase it